internal/pkg/deploy/cloudformation/cloudformation_test.go

```diff
@@ -817,6 +866,393 @@ Resources:
 	require.Contains(t, buf.String(), "A DynamoDB table to store data")
 }
 
+func testDeployWorkload_OnCancelUpdateSuccess(t *testing.T, stackName string, when func(cf CloudFormation) error) {
```

Comment:
seems like a these tests have a lot of duplicated code - is it possible to move that to a helper function?

Reply:
addressed the fb in this [commit](https://github.com/aws/copilot-cli/pull/5112/commits/b89e9b2281c85e56ff4e6297cd5be04a9414d503)